Main/ClientApp/src/Stores/CookiesUpdateStore.ts

```diff
@@ -0,0 +1,17 @@
+import ActionTypes from "../Actions/ActionTypes";
+import StoreBase from "../Infrastructure/StoreBase";
+import Action from "../Infrastructure/Action";
+
+class CookiesUpdateStore extends StoreBase {
```

Comment:
I know we wanted to use flux as much as possible but this really isn't its intened use cases. Stores are used to store data but nowthing is stored here.
For sharing the theme between components, context should be used instead of passing and listening for events (https://reactjs.org/docs/context.html)

Reply:
i'll try to fix this